languages/js/src/index.ts

```diff
@@ -3,3 +3,4 @@ export { Variable } from './Variable';
 export { AuthorizationError, ForbiddenError, NotFoundError } from './errors';
 export { Relation } from './dataFiltering';
 export { defaultEqualityFn } from './helpers';
+export { Class } from './types';
```

Comment:
Is this okay?
```suggestion
export type { Class } from './types';
```

Reply:
Also, if we're making it a public export, we should add/update the doc string on the type: https://github.com/osohq/oso/blob/054b3ddae78a1f02e26fc74090a15ca95f4c493c/languages/js/src/types.ts#L316-L320

Reply:
I have no idea what the difference is between `export` and `export type`, can you explain